bindings/python/tests/test_core.py

```diff
@@ -19,39 +19,43 @@
 
 def test_blocking():
     op = opendal.Operator("memory")
-    op.write("test", b"Hello, World!")
-    bs = op.read("test")
+    op.create_dir("test/")
+    op.write("test/test", b"Hello, World!")
+    bs = op.read("test/test")
     assert bs == b"Hello, World!", bs
-    meta = op.stat("test")
+    meta = op.stat("test/test")
     assert meta.content_length == 13, meta.content_length
 
 
 @pytest.mark.asyncio
 async def test_async():
     op = opendal.AsyncOperator("memory")
-    await op.write("test", b"Hello, World!")
-    bs = await op.read("test")
+    await op.create_dir("test/")
```

Comment:
It's a bit odd that `op.create_dir("test")` fails with

```python
>       await op.create_dir("test")
E       opendal.Error: NotADirectory (permanent) at create_dir, context: { service: memory, path: test } => read path is not a directory
```

Reply:
Should be address in https://github.com/datafuselabs/opendal/issues/1535